packages/actor-query-operation-construct/lib/BindingsToQuadsIterator.ts

```diff
@@ -74,6 +75,9 @@ export class BindingsToQuadsIterator extends MultiTransformIterator<Bindings, RD
   public static localizeBlankNode(blankNodeCounter: number,
     term: RDF.Term): RDF.Term {
     if (term.termType === 'BlankNode') {
+      if (term instanceof BlankNodeBindingsScoped) {
```

Comment:
Do we really need to preserve `BlankNodeBindingsScoped` in the context of construct queries?
AFAICS `BlankNodeBindingsScoped` is only really useful for bindings results, while we will always return quads here.

Reply:
Mh... Right, I hadn't thought of that. Will do.

Reply:
Done!